app/views/user_mailer/reservation_status_update.html.erb

```diff
@@ -0,0 +1,30 @@
+Hi <%= @reservation.reserver.name %>,
+<br />
+
+<%= render :partial => "#{@status.gsub(/\s/, '_')}" %>
+
+<br />
+
+<p><%= link_to 'Reservation Information', reservation_url(@reservation, only_path: false) %></p>
```

Comment:
Let's actually make the link text "Reservation #ID" so it's clear that it goes the reservation page.